pkgs/applications/networking/seahub/default.nix

```
@@ -4,34 +4,29 @@
 , python3
 , makeWrapper
 , nixosTests
+, seafile-server-override ? null
```

```suggestion
, seafile-server ? python3.pkgs.seaserv
```
this makes a bit more sense imo